pkg_resources/__init__.py

```diff
@@ -3470,7 +3462,7 @@ def __hash__(self):
         return self.__hash
 
     def __repr__(self):
-        return "Requirement.parse(%r)" % str(self)
+        return f"Requirement.parse({str(self)!r})"
```

Comment:
Not an automated change, but this can be simplified further

```suggestion
        return f"Requirement.parse('{self}')"
```

Reply:
I automated the detection of such explicit conversions in f-strings.